.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+quote_type = single
+
+[{deps}/**]

```

Comment:
@vinibeloni Why should it be here? I don't think there is `deps` folder somewhere